packages/components/inputs/checkbox-input/src/checkbox-input.js

```diff
@@ -31,13 +31,22 @@ const hoverStyles = (props) => {
   return css``;
 };
 
+const LabelTextWrapper = styled.div`
```

Comment:
This was added to allow showing the outline only around the option label, and not the whole container, which stretches further.